partcad-ide-vscode/src/PartcadInspector.ts

```
@@ -0,0 +1,590 @@
+     */
+    public showDone() {
+        if (this._showResolve) {
+            this._showResolve(undefined);
+        }
+    }
+
+    public async inspectPackage(pkg: ItemData) {
+        await this._view?.webview.postMessage({ type: 'package', obj: pkg, params: {} });
+    }
+
+    public async inspectSketch(sketch: ItemData, params: Object) {
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)